src/calculateRentalCost.js

```diff
@@ -4,7 +4,20 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let price = days * 40;
+  let smallDiscount = 20;
+  let bigDiscount = 50;
+  
+  if (days >= 7) {
+    return price - bigDiscount;
```

Comment:
The number '7' should be replaced with a constant that explains its meaning, such as LONG_TERM_DAYS.